### ISSUE TITLE
Failed to boot a 4.1.0 partition-based emunand

### ISSUE THREAD

**Issue by jejer** [m4xw/emuMMC#16]
Copy the bug report form Atmosphere: https://github.com/Atmosphere-NX/Atmosphere/issues/1128
### What's the issue you encountered?
Both hekete and fusee-primary failed to boot a 4.1.0 partition-based emunand emunand.
Got a black screen after Nintendo Switch Logo.
But a file-based emunand boots fine which created from the same sysnand.
**And after reboot to hekete from the file-based emunand 4.1.0, the partition-based emunand 4.1.0 could be booted successfully.**
After upgrade the partition-based emunand to 10.1.0, it boots ok.

### How can the issue be reproduced?

Create a partition and emunand using hekate from sysnand 4.1.0.
This partition-based emunand always fail to boot.
Tested on two different brands of SD cards.

### System Firmware Version

4.1.0</br>

### Environment?

- What bootloader (fusèe, hekate, etc) was Atmosphère launched by: hekate and fusee-primary
- Official release or unofficial build: 
  - Atmosphère 0.14.1
- Do you have additional kips or sysmodules you're loading: No
- Homebrew software installed: ChoiDujourNX

### Additional context?

RCM Exploitable Switch
Auto RCM disabled
Caffeine used to boot into RCM
incognito applied on sysnand
SD cards are fat32 formatted

**Comment by m4xw** [comment]
This is a nonsensical issue.
Raw based can't be broken if file based works.

**Comment by m4xw** [comment]
@jejer What happens if you rename your Nintendo folder (the emuMMC one)?

**Comment by jejer** [comment]
wow, after rename "E:\emuMMC\RAW1\Nintendo" to "E:\emuMMC\RAW1\Nintendo2" the emuMMC boots success!

**Comment by m4xw** [comment]
Probably only till a reboot, but means @CTCaer was right

**Comment by jejer** [comment]
What is the root cause, do you need to check my "E:\emuMMC\RAW1\Nintendo2"?

**Comment by m4xw** [comment]
Probably smth is up with the patching

**Comment by jejer** [comment]
yep, after N creates a new Nintendo folder, reboot goes back to a black screen.

**Comment by m4xw** [comment]
@jejer got exFAT support installed on the system?

**Comment by jejer** [comment]
No exFAT support installed.

**Comment by m4xw** [comment]
Can you try with it installed?
I wonder if both variants are broken or just the no exFAT one.
That will tell me if both need fixes or just one

**Comment by m4xw** [comment]
The actual nintendo path patches are identical, we might got something mixed up.
b9ac428bc24582aa5a2b563f800a4ac80fe65585
44ecdb457a59d365423e7bd7e312253e87b2e176

**Comment by jejer** [comment]
sure, can I just update to the same 4.1.0 with exFAT?

**Comment by m4xw** [comment]
yep

**Comment by jejer** [comment]
exFAT doesn't help, it boots to the same black screen.
Remove Nintendo directory makes the boot pass.

Both fat32 and exFAT file system on SD fail to boot for exFAT supported emuMMC.

**Comment by m4xw** [comment]
Thanks

**Comment by roostercactus** [comment]
Hi, same behavior on sysnand 4.0.1.  After setting up fat32 emunand (4.0.1) on a partition I get the black screen after the Switch logo.  To fix it's the same as well.  Power off, delete the emunand Nintendo folder, power on...now boots (only once though, if you don't upgrade firmware you have to delete the Nintendo folder on every reboot).  I updated emunand to 8.1.0 and now it's fine.  Just wanted to let you know of another firmware version where this behavior is observed.  Workaround is easy though, so it's not a big deal (as long as you know about the workaround!  It took me 2-3 days of futzing around before I found this issue).  Thanks so much for this software, it's great.

**Comment by jejer** [comment]
Tried again with hekate 5.6.5 and atmosphere 1.2.5.
The rename/remove Nintendo workaround doesn't work anymore.

After removing Nintendo, have to use atmosphere 0.19.2 with fusee-primary.bin as a middle version to boot and update the emummc from 4.1.0 to 13.1.0.

**Comment by Bidino** [comment]
I just mod with latest Atmosphere 1.2.6 + hekate v5.7.0 + Nyx v1.2.0 ecc on FW 13.2.1.
SD-based emunand is KO
file-based emunand is OK
Removing the folder every restart is OK (not praticable)

Is there any fix or workaround I miss?
Thank you

_Additional tests: reboot various times (injecting fusee.bin) result in OK or KO in a randomic way (!?)_

**Comment by m4xw** [comment]
is this actually still a issue?

**Comment by jejer** [comment]
Let me test it on the latest Atomsphere tomorrow.

**Comment by jejer** [comment]
Hi @m4xw, partition based 4.1.0 emummc boot success on Atmosphere 1.11.1 + hekate 6.5.2. Thank you!
